spec/models/subscription_spec.rb

```diff
@@ -1,5 +1,20 @@
 require 'rails_helper'
 
 RSpec.describe Subscription, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  it 'attributes cannot be in blank' do
+    subscription = Subscription.new
+
+    refute subscription.valid?
```

Comment:
Essa sintaxe não é muito usada no rspec, o que acham de:

```suggestion
    expect(subscription).not_to be_valid
```

Reply:
Tentei assim e acabou dando crash

Subscription name should be uniq
     Failure/Error: expect(subscription).not_to be_valid?
       expected #<Subscription id: nil, name: "Jornada Web com Rails", description: nil, created_at: nil, updated_at: nil, price: nil, token: nil> to respond to `valid??`
     # ./spec/models/subscription_spec.rb:17:in `block (2 levels) in <top (required)>'

Reply:
feito!

Reply:
Acho que essa não foi não